content/community/element.md

```diff
@@ -0,0 +1,8 @@
++++
+title = "Element"
+description = "Element is our preferred chat platform"
```

Comment:
This, too, is a little thorny... 

* Element is indeed our recommended Matrix client.
* Element/Matrix is *not* however, if you're critical, our preferred chat platform exclusively, since SSH uses Discord, LUP still uses IRC (for now....).
It is certainly a trending direction however...

Clear as mud ; )

Reply:
Ya, mainly just throwing content in there for testing :upside_down_face: 

But I appreciate the backstory/extra info, because I didn't realize IRC is still something LUP uses :sweat_smile:

Reply:
Addresses in this commit: https://github.com/JupiterBroadcasting/jupiterbroadcasting.com/pull/129/commits/69c9801e7f7dbd5c43e3285e1509e59761ac6eef

(just quoted wikipedia for neutrality (leaving open for more comments))